scripts/src/internal/services.ts

```diff
@@ -170,7 +170,7 @@ export async function paymentComplete(payment: CompletedPayment, logger: LoggerI
 	await order.save();
 
 	order.forEachContext(context => {
-		metrics.completeOrder(context.type, order.offerId, prevStatus, (order.currentStatusDate.getTime() - prevStatusDate.getTime()) / 1000);
+		metrics.completeOrder(context.type, order.offerId, prevStatus, (order.currentStatusDate.getTime() - prevStatusDate.getTime()) / 1000, payment.app_id);
```

Comment:
(reminder to self: I think there should be a single metric here with type=p2p instead of a separate earn/spend)